Core/HLE/sceUsbMic.h

```diff
@@ -16,5 +16,7 @@
 // https://github.com/hrydgard/ppsspp and http://www.ppsspp.org/.
 
 #pragma once
-
+//UWP doesn't like this for some reason
```

Comment:
sceUsbMic.c/h should be added to CoreUWP/CoreUWP.vcxproj and CoreUWP/CoreUWP.vcxproj.filters

Reply:
I just did that on master, so preferably just rebase this pull request on master.